R/as.trelliData.R

```diff
@@ -0,0 +1,588 @@
+#' @name .is_edata
+#' 
+#' @title Test if a file is an edata file
+#' 
+#' @param edata Must be a dataframe. Required. 
+#' 
+#' @return A boolean where TRUE means the file is an acceptable edata file. 
+#' @examples 
+#' \dontrun{
+#' 
+#' .is_edata(pmartRdata::lipid_edata)
+#' 
+#' }
+#' @export
```

Comment:
export wanted/necessary for this function?

Reply:
Yes, for test functions.